### PR TITLE
Test helpers

### PR DESCRIPTION
**Pull request by iezer**
@andyhot @rwjblue fixes a few cases for migrating test helpers like `tests/helpers/resolver.js`, and `tests/helpers/start-app.js`. Also adds a few test cases which were passing for the `config`.

### CODE REVIEW

File: lib/transforms/import-declarations.js [hunk 1]
```diff
@@ -28,6 +28,8 @@ function transformer(file, api) {
           var inTestsPrefix = appName + '/app/tests/';
           if (importPath.startsWith(inTestsPrefix)) {
             importPath = appName + '/tests/' + importPath.substring(inTestsPrefix.length);
+          } else {
+            importPath = importPath.replace('/app/app/', '/app/');
```
Comment (iezer) [inline lib/transforms/import-declarations.js]: @andyhot Not sure about the logic here, there's probably a better way to do this, but in the case that `tests/helpers/resolver.js` imports `../../app/resolver.js`, `app` gets entered twice into the path, so the path here is something like `my-app/app/app/resolver`.

Comment (andyhot) [inline lib/transforms/import-declarations.js]: Yea, there's too much magic - I hate how i hardcoded config into the makeAbsoluteHelper. I'm working on a slight refactoring that should make things saner (basically the `TODO: extract these resolutions into classic-file-info and test-file-info`) and will try your updated testcases on that

But dont wait on me if you want to move this forward

File: test/fixtures/test-helpers/input.js [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+  app: {
+    'app.js': '// app',
+    'resolver.js': '// resolver'
+  },
+  config: {
+    'environment.js': '"ENV"'
+  },
+  tests: {
+    helpers: {
+      'resolver.js': 'import Resolver from "../../app/resolver";',
```
Comment (andyhot) [inline test/fixtures/test-helpers/input.js]: Is this import really correct? From https://github.com/ember-cli/ember-cli/blob/master/blueprints/app/files/tests/helpers/resolver.js i instead see `import Resolver from '../../resolver';` which i remember i've tried and transforms correctly

Comment (andyhot) [inline test/fixtures/test-helpers/input.js]: I take that back - it doesn't currently transform correctly. But still, which of `../../app/resolver` and `../../resolver` are correct?

Comment (rwjblue) [inline test/fixtures/test-helpers/input.js]: the app/ directory is _never_ included in import paths

Comment (andyhot) [inline test/fixtures/test-helpers/input.js]: So, @iezer that explains why you had to handle the double `app`. Can you update the PR?

**Comment by iezer** [comment]
You're right about this. The file lives in `app/resolver.js` but it is imported as `resolver.js`. I've updated this PR.

@mixonic this is ready for review and merge.

**Comment by andyhot** [comment]
lgtm

**Pull request merged**